### ISSUE TITLE
ARM64 support

### ISSUE THREAD

**Issue by tudi2d** [second-state/WasmEdge-WASINN-examples#31]
Really interested in this example & would love to play around with it, but I am unfortunalley not experienced with WasmEdge yet. I wanted to start this example locally on my machine (Apple Silicon) & ran into the following problem

**Executed**: `curl -sSf https://raw.githubusercontent.com/WasmEdge/WasmEdge/master/utils/install.sh | bash -s -- --plugins wasi_nn-ggml`

**Expected**: Go through & install WasmEdge as well as the plugin

**Actual**: Installed WasmEdge, but threw an error for the specified plugin

<img width="745" alt="Screenshot 2023-09-08 at 17 18 23" src="https://github.com/second-state/WasmEdge-WASINN-examples/assets/8904624/684dbca3-5123-4126-a7b8-73cf4a81362e">

I did run the suggested `source .../.zshenv` command as well - without any luck

----
**Alternative solution**: I would also be happy to just run it in Docker, but I'm not really sure if that is possible with this plugin. Is there maybe any documentation on that? Would that be possible without being able to build the example?

**Comment by juntao** [comment]
I do not think we have ggml plugin for Mac in the release assets yet.   You can build it yourself though. But Mac needs special compiler flags in order to take advantage of the Apple neural chips. 

Please give us a couple of days to clean it up. Thanks!

**Comment by tudi2d** [comment]
Thanks for picking up this work & documenting the future work in this issue https://github.com/WasmEdge/WasmEdge/issues/2886! It should be fixed by https://github.com/WasmEdge/WasmEdge/pull/2871 when using the latest version. Will give it a try & re-open in case I run into an issue.
